src/xdocs/google_style.xml

```diff
@@ -1516,7 +1516,7 @@
                             <td><a name="7.3.4"/><a href="#7.3.4"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s7.3.4-javadoc-non-required">7.3.4 Non-required Javadoc</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s7.3.4-javadoc-non-required">7.3.4 Non-required Javadoc</a>
                             </td>
```

Comment:
This is a new paragraph (funny, there is no 7.3.3): it is a cut and past of the introductory text in 7.3 of the old specs. Additionally, it says the non-Javadocs are free from the formatting rules for Javadocs (though that would be recommended).